app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class_level: int,
+                 clean_mark_score: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class_level
+        self.clean_mark = clean_mark_score
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power_level: int,
+                 average_service_rating: float,
+                 total_service_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power_level
+        self.average_rating = average_service_rating
+        self.count_of_ratings = total_service_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+            self.wash_single_car(car)
+        return round(income, 1)
```

Comment:
Clean Code issue: You have a function `serve_cars` that does two things: it calculates the income and it washes the cars. According to the Single Responsibility Principle (SRP), a function should do one thing and do it well. It would be better to separate these two operations into two different functions. For example, you could create a separate `calculate_income` function that goes through the list of cars and calculates the total income, and a `wash_cars` function that goes through the list of cars and washes them.

Reply:
якщо я видалю рядок 27, то виникає помилка
![image](https://github.com/user-attachments/assets/c7dea127-efac-4297-b369-016a7e6df42b)